bin/mmdb_utils.py

```diff
@@ -129,21 +151,39 @@ def __init__(self, session_key):
         if not os.path.exists(LOOKUP_DIR_LOCATION):
             os.makedirs(LOOKUP_DIR_LOCATION)
 
-        account_id = self.get_max_mind_account_id()
-        if not account_id:
+        mmdb_config = self.get_max_mind_config()
+        if not mmdb_config:
             msg = "Max Mind Account key not found. Please update config from Max Mind database configuration page."
             logger.error(msg)
             raise Exception(msg)
 
         # Read MaxMind license key
         license_key = self.get_max_mind_license_key()
         if not license_key:
-            msg = "Max Mind license key not found in password store. Please update config from Max Mind database configuration page.."
+            msg = "Max Mind license key not found in password store. Please update config from Max Mind database configuration page."
             logger.error(msg)
             raise Exception(msg)
 
+        # Read MaxMind license key
+        proxy_url = None
+        try:
+            proxy_url = self.get_max_mind_proxy_url()
+        except Exception as e:
+            logger.info("Exception in getting proxy_url. {}".format(e))
+
+        if not proxy_url or proxy_url == '******' or proxy_url.lower() == 'none':
+            proxy_url = None
+            msg = "Max Mind proxy_url not found in password store. Using no proxy."
+            logger.info(msg)
+        else:
+            logger.info("Using proxy_url provided by the user.")
+
+        # SSL certificate validation
+        is_ssl_verify = convert_to_bool_default_true(mmdb_config['is_ssl_verify'])
+        logger.info("Max Mind is_ssl_verify={}".format(is_ssl_verify))
+
         # Download mmdb database in a appropriate directory
-        self.download_mmdb_database(account_id, license_key)
+        self.download_mmdb_database(mmdb_config['account_id'], license_key, proxy_url, True)
```

Comment:
need to pass is_ssl_verify instead of True

Reply:
nice catch, thanks!! Resolved.

Reply:
nice catch, thanks. Resolved.